JobConfig/cosmic/cosmic_s2_resampler.fcl

```diff
@@ -0,0 +1,48 @@
+// Re-sample cosmics from stage-1
+// Yuri Oksuzian, 2020
+
+#include "JobConfig/primary/primary.fcl"
+#include "JobConfig/cosmic/epilog.fcl"
+
+// Bring in info from the previous stage
+physics.filters.dsResample: {
+   module_type: ResamplingMixer
+   fileNames: @nil  
+   readMode: "sequential"
+   wrapFiles: true
+   mu2e: {
+      products: {
+	 genParticleMixer: { mixingMap: [ [ "generate", "" ] ] }
+	 simParticleMixer: { mixingMap: [ [ "cosmicFilter", ""] ] } 
+	 mcTrajectoryMixer:{ mixingMap: [ [ "cosmicFilter", ""] ] }
+	 stepPointMCMixer: { mixingMap: [ [ "cosmicFilter:crvStage1", ":" ], [ "cosmicFilter:CRV", ":" ] ] }
+      }
+   }
+}
+// Point Mu2eG4 to the pre-simulated data
+physics.producers.g4run.MultiStageParameters: {
+   simParticleNumberOffset: 300000  # FIX ME!!!
+   genInputHits:  [ "dsResample:crvStage1" ]
+   inputSimParticles: "dsResample"
+   inputMCTrajectories: "dsResample"
+   inputPhysVolumeMultiInfo: "" 
+}
+// Kill cosmics outside of the CRV region
+physics.producers.g4run.Mu2eG4CommonCut:{
```

Comment:
Is there a reason Mu2eG4CommonCut isn't defined in prolog?

Reply:
Same reason